scripting/tf2utils.sp

```diff
@@ -480,6 +486,21 @@ int Native_EquipPlayerWearable(Handle plugin, int numParams) {
 	}
 }
 
+// void(int wearable, bool alwaysValid);
+int Native_SetWearableAlwaysValid(Handle plugin, int numParams) {
+	int wearable = GetNativeCell(1);
+	if (!IsEntityWearable(wearable)) {
+		ThrowNativeError(SP_ERROR_NATIVE, "Entity index %d is not a wearable",
+				EntRefToEntIndex(wearable));
+	}
+
+	bool alwaysValid = GetNativeCell(2);
+	
+	Address wearableAddr = GetEntityAddress(wearable);
+
+	StoreToAddress(wearableAddr + offs_CEconWearable_bAlwaysValid, alwaysValid, NumberType_Int8);
```

Comment:
Swap this out with `SetEntData(wearable, offs_CEconWearable_bAlwaysValid, alwaysValid, 1)`; `StoreToAddress` is a relatively slow operation especially the tooling currently targets 1.10.

Reply:
i forgor about SetEntData